src/native/libs/System.JavaScript/ReadMe.md

```diff
@@ -0,0 +1,15 @@
+# System.JavaScript
+
+Is TypeScript project that is compiled via `src\native\libs\rollup.config.js` into Emscripten JS library.
+And a native library compiled by CMake into `System.JavaScript.a` as part of `/src/native/libs/CMakeLists.txt`
+
+## System.JavaScript.ts -> dotnet.native.js
+
+This is root of **Emscripten library** that would become part of `dotnet.runtime.js`
+It implements native parts of PAL for the VM/runtime.
```

Comment:
I would lead start the readme with the description of what the directory contains like "It implements native parts of PAL for the VM/runtime.". It is the most important information that somebody not familiar with the project needs. How does it get built is secondary.